typedb/connection/cluster/server_client.py

```diff
@@ -63,7 +63,6 @@ def _new_channel(self) -> grpc.Channel:
 
     def close(self) -> None:
         super().close()
-        self._channel.close()
```

Comment:
See PR description for the full explanation of why we do this. In brief, explicitly calling `close` on a gRPC Channel in Python can result in resources being deallocated while they are still being used, which in turn may cause gRPC's native C libraries to segfault. Not closing the channel is the approach recommended in an issue in the gRPC repository itself.